app/build.gradle.kts

```diff
@@ -5,10 +5,13 @@ plugins {
     id("org.jlleitschuh.gradle.ktlint")
     id("uk.gov.onelogin.jvm-toolchains")
     id("com.google.dagger.hilt.android")
-
+    id("io.gitlab.arturbosch.detekt")
```

Comment:
I can see that the id-check project was using detekt, so I am assuming that's why you added it to this project as well.
But I was just curious if we decided to use detect/sonar going forward? or can we use both?

Reply:
Think we actually use both on Id check currently, so can discuss if that's something we want to do or not